deployment/docker/Makefile

```diff
@@ -169,15 +169,26 @@ package-via-docker-386:
 remove-docker-images:
 	docker image ls --format '{{.Repository}}\t{{.ID}}' | awk '{print $$2}' | xargs docker image rm -f
 
+# docker-single-up:
+# 	docker-compose -f deployment/docker/docker-compose.yml up
+
+# docker-single-down:
+# 	docker-compose -f deployment/docker/docker-compose.yml down -v
+
+# docker-cluster-up:
+# 	docker-compose -f deployment/docker/docker-compose-cluster.yml up
+
+# docker-cluster-down:
+# 	docker-compose -f deployment/docker/docker-compose-cluster.yml down -v
 
 docker-single-up:
-	docker-compose -f deployment/docker/docker-compose.yml up
+	docker compose -f deployment/docker/docker-compose.yml up -d
```

Comment:
Wdyt about moving `docker compose` into a make variable?
That would allow users who are using older version or want to use compose V1 to use something like:
```
make docker-single-up DOCKER_COMPOSE=docker-compose
```